provisioning/vars/mangaki.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+mangaki_git_repository: "https://github.com/mangaki/mangaki"
+mangaki_root: "/mangaki"
+mangaki_venv: "{{ mangaki_root }}/.venv"
+mangaki_repo: "{{ mangaki_root }}/app"
+mangaki_app: "{{ mangaki_repo }}/mangaki"
+
+discourse_sso_secret: ''
+discourse_api_username: 'JJ'
+discourse_api_key: ''
+
+mal_user: ''
+mal_pass: ''
+mal_user_agent: ''
```

Comment:
To be honest, oui.
Mais ça me perturbe ces variables.

C'est chiant qu'on puisse pas configurer ces deux sets de service facilement.
Déjà, `MAL_USER_AGENT` est-il vraiment nécessaire? :/
Je pense qu'on peut default à `mangaki/crawler`